.github/workflows/test-example-native.yml

```diff
@@ -0,0 +1,37 @@
+name: Test Example Native
+
+on: [workflow_dispatch]
+
+jobs:
+  test-native:
+    name: Run perf test in Native
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.x
+          cache: 'yarn'
+
+      - name: Validate TypeScript
+        run: yarn typecheck
```

Comment:
Use `yarn --cwd examples/native [command]` for all commands to run them in the proper folder.